lucid-book-store/src/components/cart-wishlist/cart-operations.jsx

```diff
@@ -0,0 +1,68 @@
+import { useCart } from "../../context/cart-wishlist-context";
+import { checkInList } from "../../utility/check-in-list";
+
+const AddToCart = ({ product }) => {
+  const { cartState, cartDispatch } = useCart();
+  const clickHandler = () => {
+    if (checkInList(cartState.cart, product._id)) {
```

Comment:
I am guessing this line of code increases quantity if click again on add_to_cart button, if it is true then it's a good feature :)

Reply:
Yes if the product is in the cart then clicking on add to cart will just increase it's quantity